documents/store-operations/bopis/order-page/gift-card-activation.md

```diff
@@ -0,0 +1,28 @@
+# Gift Card Activation
+
+The BOPIS App allows store staff to activate gift cards directly while fulfilling orders. This helps in handling gift card items without switching to any other system.
+
+## When Does the Gift Card Activation popup Appear?
+
+The activation pop-up opens when a staff member taps on a gift card item from the orders list.
+
+## Steps to Activate a Gift Card
+
+**Step 1: Enter the Activation Code**  
+The staff member can scan or manually type the serial number printed on the physical gift card. It’s important to make sure the code is correct before moving forward.
+
+**Step 2: Tap the Activate Button**  
+After entering the code, tapping the Activate button (represented by a gift card icon in the corner) will bring up a confirmation message. This is the final opportunity to review the entered details.
+
+**Step 3: Confirm Activation**  
+Tapping Activate again on the confirmation popup completes the process. If activation is successful, a confirmation message appears, and the gift card number is displayed along with the activation timestamp.
+
+## Important Notes
+
+- Once a gift card is activated, the details can’t be changed.  
+- If activation doesn’t go through, an error message will appear. It helps to recheck the code and try again.  
+- Some stores may notify customers after activation, so it’s good practice to review everything carefully before confirming.
```

Comment:
does this have anything to do with the process? what did the user learn by reading this? I think we can delete it?